index.js

```diff
@@ -1,9 +1,9 @@
-import bookData from './modules/Books';
+import Books from './modules/Books.js';
```

Comment:
https://github.com/HeDevedUp/wireframeEs6/blob/461576febc4f559ca5256274793f4ee39b8f336a/modules/list_books.js#L3

- [ ] You should use the ``` arrow``` function for this project, it's a requirement to use ES6 syntax, please change all the ```functions``` to ``` arrow functions```